utam-compiler/src/test/java/utam/compiler/grammar/TestUtilities.java

```diff
@@ -103,14 +94,5 @@ private static JsonNode createStringTypeNode(String type) {
       }
       return new TextNode(type);
     }
-
-    private static JsonNode createArrayTypeNode(String[] type) {
-      if (type == null) {
-        return null;
-      }
-      ArrayNode node = new ArrayNode(JsonNodeFactory.instance);
-      Arrays.stream(type).forEach(node::add);
-      return node;
-    }
```

Comment:
those methods are no longer used in tests because tests were replaced by reading JSON file instead